packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/notification/NotificationEvent.java

```diff
@@ -64,11 +64,6 @@ protected void onCreate() {
 	@Column(columnDefinition = "text")
 	private JsonNode data;
 
-	public NotificationEvent setData(final JsonNode arg) {
```

Comment:
What happens now if we pass JsonNode to set data? I thought without the override it would choke on `valueToTree`

Reply:
Disregard, it works fine.

Reply:
:) - I only saw the test using that function, and as tests are fake I made the test conform to the actual usage (as I saw)